### PR TITLE
add Fee

### PR DESCRIPTION
**Pull request by Xiaxuliang**
pinnako add Fee

### CODE REVIEW

**Comment by llamatester** [comment]
The pinnako.ts adapter exports: 
        
 
 _______________________________________
    Fees for 10/7/2023
    _______________________________________
    
    ZKSYNC 👇
    Backfill start time: 10/12/2022
    Timestamp: 1689033598
    Daily fees: 123.0335015845749
    └─ Methodology: All mint, burn, margin and liquidation and swap fees are collected
    Daily user fees: 123.0335015845749
    └─ Methodology: Users pay swap fees and margin and liquidation fees
    Daily revenue: 36.91005047537247
    └─ Methodology: Revenue is calculated as 30% of the total fee.
    
    
    
    
    Done in 3.27s.

**Comment by llamatester** [comment]
The pinnako.ts adapter exports: 
        
 
 _______________________________________
    Fees for 10/7/2023
    _______________________________________
    
    ZKSYNC 👇
    Backfill start time: 10/12/2022
    Timestamp: 1689033598
    Daily fees: 2493.010659651163
    └─ Methodology: All mint, burn, margin and liquidation and swap fees are collected
    Daily user fees: 0.71205965116332
    └─ Methodology: Users pay swap fees and margin and liquidation fees
    Daily revenue: 747.903197895349
    └─ Methodology: Revenue is calculated as 30% of the total fee.
    
    
    
    
    Done in 3.70s.

File: fees/pinnako.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import { Chain } from "@defillama/sdk/build/general";
+import { gql, request } from "graphql-request";
+import type { ChainEndpoints } from "../adapters/types";
+import { Adapter } from "../adapters/types";
+import { ZKSYNC } from "../helpers/chains";
+import { getTimestampAtStartOfDayUTC } from "../utils/date";
+
+const endpoints = {
+    [ZKSYNC]: "https://graph.pinnako.io/subgraphs/name/ekmanss/stats",
```
Comment (dtmkeng) [inline fees/pinnako.ts]: seem tvl is set era chain chain can set `CHAIN.ERA` instead new value

**Review by dtmkeng — CHANGES_REQUESTED** [review]

File: fees/pinnako.ts (hunk 1 above)
Comment (Xiaxuliang) [reply fees/pinnako.ts]: Has changed

**Comment by llamatester** [comment]
The pinnako.ts adapter exports: 
        
 
 _______________________________________
    Fees for 11/7/2023
    _______________________________________
    
    ZKSYNC 👇
    Backfill start time: 10/12/2022
    Timestamp: 1689119998
    Daily fees: 379.42811625135744
    └─ Methodology: All mint, burn, margin and liquidation and swap fees are collected
    Daily user fees: 49.064675704536675
    └─ Methodology: Users pay swap fees and margin and liquidation fees
    Daily revenue: 113.82843487540723
    └─ Methodology: Revenue is calculated as 30% of the total fee.
    
    
    
    
    Done in 4.54s.

**Comment by dtmkeng** [comment]
it should `era` not `ZKSYNC` it like tvl adapter ?

**Comment by llamatester** [comment]
The pinnako.ts adapter exports: 
        
 
 _______________________________________
    Fees for 11/7/2023
    _______________________________________
    
    ERA 👇
    Backfill start time: 10/12/2022
    Timestamp: 1689119998
    Daily fees: 379.42811625135744
    └─ Methodology: All mint, burn, margin and liquidation and swap fees are collected
    Daily user fees: 49.064675704536675
    └─ Methodology: Users pay swap fees and margin and liquidation fees
    Daily revenue: 113.82843487540723
    └─ Methodology: Revenue is calculated as 30% of the total fee.
    
    
    
    
    Done in 3.38s.

**Comment by Xiaxuliang** [comment]
> it should `era` not `ZKSYNC` it like tvl adapter ?

ok

**Comment by llamatester** [comment]
The pinnako adapter exports: 
        
 
 _______________________________________
    Dexs for 11/7/2023
    _______________________________________
    
    Version -> SWAP
    ---------
    ERA 👇
    Backfill start time: 5/7/2023
    NO METHODOLOGY SPECIFIED
    Timestamp: 1689033600
    Daily volume: 3841.661099965036
    Total volume: 5543.937223738246
    
    
    Version -> DERIVATIVES
    ---------
    ERA 👇
    Backfill start time: 5/7/2023
    NO METHODOLOGY SPECIFIED
    Timestamp: 1689033600
    Daily volume: 24438.39108501348
    Total volume: 187689.375826824
    
    
    Done in 4.39s.

**Comment by llamatester** [comment]
The pinnako.ts adapter exports: 
        
 
 _______________________________________
    Fees for 11/7/2023
    _______________________________________
    
    ERA 👇
    Backfill start time: 10/12/2022
    Timestamp: 1689119998
    Daily fees: 379.42811625135744
    └─ Methodology: All mint, burn, margin and liquidation and swap fees are collected
    Daily user fees: 49.064675704536675
    └─ Methodology: Users pay swap fees and margin and liquidation fees
    Daily revenue: 113.82843487540723
    └─ Methodology: Revenue is calculated as 30% of the total fee.
    
    
    
    
    Done in 3.77s.

**Pull request merged**